frontend/national-boxing-championships-explorer/index.html

```
@@ -0,0 +1,685 @@
+            <option value="vijender-singh">Vijender Singh</option>
+            <option value="nikhat-zareen">Nikhat Zareen</option>
+            <option value="lovlina-borgohain">Lovlina Borgohain</option>
+            <option value="amit-panghal">Amit Panghal</option>
+            <option value="shiva-thapa">Shiva Thapa</option>
+            <option value="sarita-devi">Sarita Devi</option>
+            <option value="manish-kaushik">Manish Kaushik</option>
+          </select>
+        </div>
+
+        <div id="boxer-profile" class="boxing-boxer-profile" aria-hidden="true" style="display: none;">
+          <img id="boxer-profile-img" src="" alt="" loading="lazy">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)